package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-sankey",
-  "version": "3.1.3.0",
+  "version": "3.2.0",
```

Comment:
Return 4 digit version. It's mandatory for all CVs

Reply:
done